00-programming-fundamentals/exercises/merge-conflict-remote.md

```diff
@@ -0,0 +1,102 @@
+
+## Learning Goals
+
+The goal of this activity is to give you exposure to the process of resolving merge conflicts. Because merge conflicts are a natural part of any collaborative development effort, this is an important skill to have as a developer.
+
+With this activity you will be able to encounter and resolve merge conflicts in a controlled scenario where the correct resolution of all merge conflicts is clear.
+
+## Instructions
+
+### Baseline Setup
+
+Before we can start putting the recipe together we need to make sure that everyone in your group is working from the same starting point and can effectively contribute to the recipe building effort.
+
+1. **Split into groups**: The class will be divided into groups using the breakout sections feature of Zoom, these assignments are random.
+   1. Breakout Room 1:  Aardvark
+   2. Breakout Room 2:  Tarsier
+   3. Breakout Room 2:  Gecko
+   4. Breakout Room 2:  Sloth
+   5. Breakout Room 2:  Lionfish
+   6. Breakout Room 2:  Pangolin
+   7. Breakout Room 2:  Rhino
+   8. Breakout Room 2:  Curassow
+2. **Fork the activity repo**: One person in your group should fork the carnival-recipes repo and then add everyone else as a collaborator.
+3. **Clone the forked repo**: Everyone else in your group should clone the forked repository.
+4. **Find your recipe scrap**: Each group is assigned six recipe scraps that should all be part of the same recipe. You can split these scraps up amongst your group however you wish. The scraps assigned to each group are listed in the carnival-recipes repo. If your group has fewer than six people someone may need to take multiple scraps.
+5. **Add your recipe scrap**: Copy the contents of your recipe scrap into the recipe file and then add and commit your changes to the recipe file. After completing this step each person in your group should have **different** content within the recipe file in their local repository.
+
+**Do not** push your changes to GitHub yet! If you're finished early with this step help your groupmates.
+
+### Building the recipe
+
+Once everyone in your group has completed the baseline setup instructions above you're ready to start building the recipe together.
+
+#### General advice
+
+There are a few things to look out for when working through this activity:
+
+- If you're still unsure about what a merge conflict is, or how to resolve one please review the [Merge Conflicts curriculum](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/git-merge-conflicts.md).
+- You can review the [slides](https://docs.google.com/presentation/d/15OFyyoNJevzk7IpdZCP6VwMLju0MVS2pv0qeETLOa6Q/edit?usp=sharing) and [example walkthrough video](https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=ec7dbf60-e59b-4317-a4a5-abbe002eb7ea) if you are unsure about how to complete the activity.
```

Comment:
I appreciate you making an example video as well. I'm a little confused about how we expect they'd utilize this.
I wonder if we should either (1) assign this as a video to watch before the activity or (2) suggest that if they get stuck, one of the students do a screen share so that they can all watch the video together? But also the intention is that 2 or 3 instructors will be popping between the breakout rooms to support, right? If so, that feels to me like probably a more effective way to clarify things than asking the whole group to stop and watch a video...so I'm less sold on option 2. 
And also not sure that option 1 is necessary either. 
But curious for other thoughts

Reply:
If they get confused they can refer to the video, I'm not sure it's needed, but included it to parallel the original assignment which included it.

Reply:
I'm just hesitant to include something like a video in this group context because I fear it could cause confusion. ("Are we all supposed to watch this right now? Have we seen this video before? Should we all watch it separately and come back together?")
Would you be up for just taking it out and counting on the rotating instructors to provide guidance instead of the video?